wiki/Gameplay/Full_combo/en.md

```diff
@@ -6,10 +6,21 @@ tags:
 
 # Full combo
 
-**Full combo** (or *FC* for short) is a term used to describe a player obtaining the maximum possible [combo](/wiki/Beatmapping/Combo) on a [beatmap](/wiki/Beatmap). Full combos are scored by passing a beatmap with no misses<!-- TODO: link -->, [sliderbreaks](/wiki/Gameplay/Judgement/Slider_break), or dropped [slider ends](/wiki/Gameplay/Hit_object/Slider/Slidertail).
+**Full combo** (or *FC* for short) is a term used to describe a player passing a [beatmap](/wiki/Beatmap) without ever combo breaking. Full combos are scored by passing a beatmap with no misses<!-- TODO: link --> or [sliderbreaks](/wiki/Gameplay/Judgement/Slider_break).
 
 Scores that lost combo only via dropped slider ends are widely considered by the community to be full combos. This differs from the game client and website's display.
 
 ![Screenshot of full combo and broken combo scores](img/combo-comparison.png "The top score is a full combo, and the bottom score is a broken combo.")
 
 Because the [score](/wiki/Gameplay/Score) of an individual object in [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) heavily depends on combo multiplier and is virtually unbounded, full combos typically award the most score in these game modes.
+
+# Perfect full combo
+**Perfect full combo** (or *PFC* for short) is a term used to describe the game acknowledging a full combo ocurred by coloring the combo green. This is crucial for Hush-Hush medals, as whenever they are checking for the full combo conditional directly, it is referring to a perfect full combo. In all 4 game modes, Standard, Taiko, Catch, and Mania, the way this is technically processed varies:
+
+Standard - Pass a beatmap with no misses, sliderbreaks, or dropped slider ends.
+
+Taiko - Pass a beatmap with no misses. The beatmap must not contain sliders/drumrolls or spinners/dendens/shakers. Taiko skill medals however allow sliders/drumrolls or spinners/dendens/shakers.
+
+Catch - Pass a beatmap with no misses.
+
+Mania - Pass a beatmap with no misses, 50s, or 100s. Mania skill medals however allow 100s.
```

Comment:
Mania is the most chaotic when it comes to a PFC (perfect full combo).

Other than misses, if a player gets at least one 50 or one 100 in a mania score, **they have ruined their PFC**. I am just stating how mania sees it.

The best example is those that go for the Hush-Hush medal called (spoilers: clicking this hyperlink will reveal its solution to you) [Mirage](https://osekai.net/medals/?medal=Mirage). If you check out the mania leaderboards in that particular map, you will see that the only players that get the medal are the ones who were successfully able to make their combo green, otherwise known as having achieved a perfect full combo. How did they do that? They got 0x misses, 0x 50s, and 0x 100s. That is just how mania is, and it EXTREMELY confusing and frustrating.